apps/opencs/model/tools/pathgridcheck.cpp

```diff
@@ -29,6 +21,12 @@ int CSMTools::PathgridCheckStage::setup()
 
 void CSMTools::PathgridCheckStage::perform (int stage, CSMDoc::Messages& messages)
 {
+    // NOTE: This is horribly inefficient but in order to use signals the entire Stage class
+    // hierarchy needs to be braught under Qt which seems like an overkill for a small
+    // performance gain during verify operations
+    CSMSettings::UserSettings &userSettings = CSMSettings::UserSettings::instance();
```

Comment:
Ok - so not possible to use settings at all?  I tried putting the user setting in the constructor but means a restart before a new setting will take hold.